docs/Grade.md

```diff
@@ -0,0 +1,58 @@
+# Grade Component
```

Comment:
You may want to polish section titles here to match the readmes of our other components. The [documentation of Hero & Hero Banner](https://github.com/etn-ccis/blui-react-native-component-library/blob/dev/docs/Hero.md) is the closest to what you are writing here.